README.md

```diff
@@ -53,35 +56,32 @@ There are three groups of tests:
 
 * Unit tests
   * Simple unit tests for individual functions and classes.
-  * **Framework:** Mocha
-  * **Assertions:** Chai
+  * **Framework:** [Vitest](https://vitest.dev/)
+  * **Assertions:** [Chai](https://www.chaijs.com/)/[Jest](https://jestjs.io/docs/expect)
   * **Path:** `tests/unit`
-  * **Command:** `yarn run test:unit`
-  * (To watch add `--watch`)
+  * **Command:** `yarn run test:unit` (watches by default, only re-runs changed file)
+  * (To prevent watching, use `yarn vitest run`)
 * Component tests
   * In-browser tests which mount a single Vue component standalone.
-  * **Framework:** Cypress
+  * **Framework:** [Cypress](https://docs.cypress.io/guides/overview/why-cypress)
   * **Assertions:** Chai
   * **Path:** `cypress/component`
-  * **Command:** `yarn run test:e2e` (navigate to component page)
-  * (For "headless" mode add `-- --headless --config video=false`)
+  * **Command:** `yarn run test:component`
+  * (For "headless" mode use `yarn cypress run --component --config video=false`)
```

Comment:
I think probably these last lines need to be continuation of the `**Command**` bullet point. Not militant about this though.